apiserver/plane/authentication/utils/host.py

```
@@ -1,18 +1,16 @@
-
 # Django imports
 from django.conf import settings
+from django.http import HttpRequest
+# Third party imports
+from rest_framework.request import Request
 
+# Module imports
+from plane.utils.ip_address import get_client_ip
 
-def base_host(request, is_admin=False, is_space=False, is_app=False):
+def base_host(request: Request | HttpRequest, is_admin: bool = False, is_space: bool = False, is_app: bool = False) -> str:
```

Ruff (0.8.2)

Line too long (123 > 88)

(E501)